### PR TITLE
Added usage tip and LRW explanation

### PR DESCRIPTION
**Pull request by talogan**

### CODE REVIEW

**Review by Jlrine2 — APPROVED** [review]

File: asf_tools/README.md [hunk 1]
```diff
@@ -45,6 +45,13 @@ To generate a composite of the cross-polarization images, navigate to the direct
 make_composite VH-composite */*VH.tif
 ```
 
+#### Usage Tip
+Because the imagery has been radiometrically terrain corrected (RTC), geometric and radiometric distortions have been removed from the files to be mosaicked.  One the strong points of LRW composite mosaics is that you combine both ascending and descending datatakes into a singe mosaic. In this manner no layover or shadow masks are required - what is shadowed on an ascending pass is visible in a descending pass and vice-versa.  Thus, not only is it possible to combine ascending and descending, but it is highly encouraged.  Using many datatakes from both the ascending and descending satellite passes will make the best mosaics possible.
```
Comment (jhkennedy) [inline asf_tools/README.md:49]: mosaicked --> composited
mosaics --> composites x2
mosaic --> composite

To be inline with how D. Small discusses these products and the rest of the documentation.

File: asf_tools/README.md [hunk 2]
```diff
@@ -45,6 +45,13 @@ To generate a composite of the cross-polarization images, navigate to the direct
...
+
+#### About Local Resolution Weighting (LRW)
+In an LRW composite, each satellite pass contributes to creating the output pixels.  The amount of this contribution is scaled by the inverse of the scattering area used during terrain correction.  (Thus the need for requesting the area_map option of HyP3 RTC).  The inverse of the RTC scattering area, also referred to as local resolution, is multiplied by each pixel's backscatter value.  The results of all of the images covering any single pixel are then summed.  This total is then divided by the sum of the weights used to get the output average backscatter.
```
Comment (jhkennedy) [inline asf_tools/README.md:52]: Parenthetical "(Thus...) should be moved inside the previous sentence (preffered), or it's concluding period move inside the parentheses. 

"The inverse of the RTC scattering area" -- RTC feels weird here, I think "surface scattering area" might be better. 

"are then summed.  This total"  reads a bit better like "are summed and the total"

File: asf_tools/README.md (hunk 1 above)
Comment (talogan) [reply asf_tools/README.md]: Done.  Not even gonna let me sneak in a single mosaic...?  Geez.

File: asf_tools/README.md (hunk 2 above)
Comment (talogan) [reply asf_tools/README.md]: Agreed, done.

File: asf_tools/README.md (hunk 1 above)
Comment (jhkennedy) [reply asf_tools/README.md]: haha!

**Review by jhkennedy — APPROVED** [review]

File: asf_tools/README.md (hunk 2 above)
Comment (jhkennedy) [reply asf_tools/README.md]: Looks good! Approved!

**Pull request merged**